_pages/opportunities.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Opportunities
+data: opportunities
+---
+
+<div id = "opportunities-page">
+  <div id="opportunities">
+  </div>
+</div> 
+
+<script src="./papaparse.min.js"></script>
+<script src="./moment.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.3.1.js"></script>
+  
+  <script>
+    const link = 'https://docs.google.com/spreadsheets/d/e/2PACX-1vSNnb-Ca4JiuC3OuX7W2IgF1SFSsDgELWJjNN1vgaHemw7RjAYHky3AGre2t5QGKRSD9jZ4SdnznmE1/pub?output=csv';
+
+    function init() {
+      Papa.parse(link, {
+      download: true,
+      header: true,
+      complete: function(results) {
+        var data = results.data
+        let date = moment();
+        let eventDate;
+        let closingDate;
+        let closingMessage;
+
+        
+        
+        if (data.length === 0) {
+          $('#opportunities').append(
+            '<div style="text-align: center">' +
+              '<h5 class="opp-title"><span>No opportunities available at this time</span></h5>' +
+            '</div>'
+          );
+        } else {
+          console.log(data.length);
+          for (var i = 0; i < data.length; i++) {
+
+            if (data[i].ClosingDate) {
+              closingDate = moment(data[i].ClosingDate, "DD-MM-YYYY").format("Do MMMM YYYY")
+              closingMessage = '<p class="opp-title"><b>Applications close on '+ closingDate + '</b></p>'
+            }
+
+            $('#opportunities').append(
```

Comment:
Could we have different styling for different types of opportunities, like: 
- Online courses
- Internship
- Study abroad? 
- Local work at uni 
- Volunteering
Something like this?